apps/web/playwright/webhook.e2e.ts

```diff
@@ -428,7 +421,7 @@ test.describe("FORM_SUBMITTED", async () => {
     expect(page.locator(`text='${webhookReceiver.url}'`)).toBeDefined();
 
     await page.waitForLoadState("networkidle");
-    await page.goto("/apps/routing-forms/forms");
```

Comment:
Though this URL would still work, changed the reference to something that would be normally accessed.